models.py

```diff
@@ -1,33 +1,33 @@
 from sqlalchemy import ForeignKey
 
-from envVariables import db
+from variables import db
 
 
 class UserModel(db.Model):
     __tablename__ = 'users'
     email = db.Column(db.String(20), primary_key=True)
     name = db.Column(db.String(30), primary_key=True)
-    phoneNumber = db.Column(db.String(10), nullable=False)
+    phone_number = db.Column(db.String(10), nullable=False)
     city = db.Column(db.String(20), nullable=False)
     street = db.Column(db.String(20), nullable=False)
     password = db.Column(db.String(50), nullable=False)
-    isAdmin = db.Column(db.Boolean, nullable=False)
+    is_admin = db.Column(db.Boolean, nullable=False)
 
 
 class DoneeModel(db.Model):
     __tablename__ = 'donees'
     id = db.Column(db.String(20), primary_key=True)
-    fName = db.Column(db.String(15), nullable=False)
-    lName = db.Column(db.String(15), nullable=False)
+    first_ame = db.Column(db.String(15), nullable=False)
+    last_name = db.Column(db.String(15), nullable=False)
     city = db.Column(db.String(20), nullable=False)
     street = db.Column(db.String(20), nullable=False)
-    phoneNumber = db.Column(db.String(10), nullable=False)
+    phone_number = db.Column(db.String(10), nullable=False)
 
 
 class DonationModel(db.Model):
     __tablename__ = 'donations'
```

Comment:
A `UniqueConstraint` should be defined in the `__table_args__` tuple to treat the 3 primary keys as a composite key.